_layouts/lesson.html

```diff
@@ -182,7 +182,9 @@ <h4>{{site.data.snippets.difficulty[page.lang]}}</h4> {% include difficulty.html
   {% if site.lesson_donation_alerts %}
   {% include support-alert.html %}
   {% endif %}
+ 
 
+{{ if translation_candidates.size >0 }} 
```

Comment:
Look at the [Liquid docs](https://shopify.github.io/liquid/tags/control-flow/) for a reminder on how to use `if` tags - you need to use `{% %}`

[Here's an approach to the code that checks](https://gist.github.com/mdlincoln/bfd32193045937ab2a7ed9da3a09d5c1):

1. Are there any translation candidates? OR Is there an original?
2. Prints the `translation_source` link first, and _then_ loops through all the `translation_candidates`